nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenTCPRecord.java

```diff
@@ -436,28 +401,23 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                     session.transfer(flowFile, REL_SUCCESS);
                 }
 
-                getLogger().debug("Re-queuing connection for further processing...");
-                socketReaders.offer(socketRecordReader);
-
+                getLogger().debug("Re-queuing reader for further processing...");
+                recordReaders.offer(recordReader);
             } catch (Exception e) {
                 getLogger().error("Error processing records: " + e.getMessage(), e);
```

Comment:
This could be adjusted to remove unnecessary repetition of the exception message and the stack trace from the the exception itself.
```suggestion
                getLogger().error("Record Processing Failed", e);
```